castle/cms/browser/templates/secure-login.pt

```diff
@@ -10,7 +10,7 @@
       tal:define="utils context/@@castle-utils;
                   site_login python: hasattr(context,'portal_registry');
                   portreg python: context.portal_registry if site_login else {'plone.backend_url':[],'plone.only_allow_login_to_backend_urls':False};
-                  backend_urls python:portreg['plone.backend_url'];
+                  backend_urls python:portreg['plone.backend_url'] or [];
```

Comment:
Does this cover all situations, e.g. empty string and None?

Reply:
While it shouldn't ever happen, if it's an empty string, leng('') will still return 0. That's fine.
If it's None, it defaults to [].